terraform/browser/browser.tf.json.template.py

```diff
@@ -48,6 +48,59 @@
 provision_custom_search = False
 
 
+def content_security_policy() -> str:
+    def q(s: str) -> str:
+        return f"'{s}'"
+
+    def s(*args: str) -> str:
+        return ' '.join(args)
+
+    self = q('self')
+    none = q('none')
+    return ';'.join([
+        s('default-src', self),
+        s('object-src', none),
+        s('frame-src', none),
+        s('frame-ancestors', none),
+        s('child-src', none),
+        s('img-src',
+          self,
+          'data:',
+          'https://lh3.googleusercontent.com',
+          'https://www.google-analytics.com',
+          'https://www.googletagmanager.com'),
+        s('script-src',
+          self,
+          q('unsafe-inline'),
+          q('unsafe-eval'),
+          'https://accounts.google.com/gsi/client',
+          'https://www.google-analytics.com',
+          'https://www.googletagmanager.com'),
+        s('style-src',
+          self,
+          q('unsafe-inline'),
+          'https://fonts.googleapis.com',
+          'https://p.typekit.net',
+          'https://use.typekit.net'),
+        s('font-src',
+          self,
+          'data:',
+          'https://fonts.gstatic.com',
+          'https://use.typekit.net/af/'),
```

Comment:
If the trailing slashes are insignificant, I'd prefer to consistently omit them. If they are significant, please explain what the effect is. You might need to confer with CC and/or look up the standard.

Reply:
Trailing slashes are significant.

https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy?ref=akshaykhot.com#hosts_values

> - Path parts in the CSP that end in `/` match any path they are a prefix of. For example, `example.com/api/` will match URLs like `example.com/api/users/new`.
> - Other path parts in the CSP are matched exactly; for example, `example.com/file.js` will match `http://example.com/file.js` and `https://example.com/file.js`, but not `https://example.com/file.js/file2.js`.